tracer/src/Datadog.Trace/Activity/DuckTypes/IActivityListener.cs

```diff
@@ -0,0 +1,24 @@
+// <copyright file="IActivityListener.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.Activity.DuckTypes
+{
+    internal interface IActivityListener : IDuckType
```

Comment:
Might be nice to comment it so we remember that later :)
```suggestion
    // Using interface instead of [DuckCopy] struct as we need to set values too
    internal interface IActivityListener : IDuckType
```

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/2446/commits/8533973cae079282f05839fc49fad6d16a526b83